content/format-reduction/index.md

```diff
@@ -0,0 +1,56 @@
+---
+rfc:
+start_date: 2018-07-26
+pr:
+status: draft
+---
+
+# Format reduction
+
+## Summary
+
+This RFC proposes to reduce the amount of representations in Registers.
```

Comment:
This is a minor thing but "in Registers" makes me think less of the presentation formats and more about the internal structure of a register and the data that can be stored in one.

I'd describe this RFC as reducing "the amount of representations a registers implementation should support"